tests/__main__.py

```diff
@@ -136,22 +188,33 @@ def run_tests(argv, env, coverage=False):
             print('...coverage failed!')
             sys.exit(rc)
         print('...done')
+    elif config.junit_xml:
+        os.environ.update(env)
+        with open(config.junit_xml, 'wb') as output:
+            unittest.main(
+                testRunner=XMLTestRunner(output=output),
+                module=None,
+                argv=argv
+            )
     else:
         os.environ.update(env)
         unittest.main(module=None, argv=argv)
 
 
 if __name__ == '__main__':
-    argv, env, runtests, lint = convert_argv(sys.argv[1:])
+    config, args, env = convert_argv(sys.argv[1:])
     fix_sys_path()
-    if lint:
+    if config.lint or config.lint_only:
         check_lint()
-    if runtests:
-        if '--start-directory' in argv:
-            start = argv[argv.index('--start-directory') + 1]
-            print('(will look for tests under {})'.format(start))
+    if not config.lint_only:
+        if config.start_directory:
```

Comment:
@ericsnowcurrently This is why I added the `--start-directory` option to argparse, as I saw that we used the value to report something down the line.  However, I did fail to then add that option back to the list of args passed through to unittest. I'll put this behaviour back the way it was, updating to look through the 'passthrough' args.